src/auth/ui/vue/telemetry.vue

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { WebviewClientFactory } from '../../../webviews/client'
+import { AuthUiClick, AuthWebview } from './show'
+
+const client = WebviewClientFactory.create<AuthWebview>()
+
+export default defineComponent({
```

Comment:
Yeah, I was using it so that other Components could extend it, allowing us to reuse the `emitUiClick()` method it defines within the HTML. Otherwise we need to define the same method for each component.